controllers/CommentController.js

```diff
@@ -267,4 +268,24 @@ export default class CommentController {
       return next(error);
     }
   }
+
+  /**
+   * Method to get  an article comments and their likes
+   *
+   * @static
+   * @param {object} req
+   * @param {object} res
+   * @param {function} next
+   * @returns {object} details of the edited comment
+   * @memberof CommentController
+   */
+  static async getArticleComment(req, res, next) {
+    const { articleId } = req.params;
+    try {
+      const commentsLike = await Like.findAll({ where: { articleId }, attributes: ['userId', 'articleId', 'like', 'commentId'] });
```

Comment:
Are you finding the likes of an article or likes of a comment? Can you explain? Maybe I don't understand how the model is structured